admin/src/pages/Order/components/OrderDetails/index.js

```diff
@@ -50,44 +56,45 @@ export default function OrderDetails() {
 
   return (
     <Layout sidebar>
-      <div className='order-detail'>
-        <h3 className='order-owner'>
-          Order by :
-          <span>
-            <strong>{orderDetails.user.userName}</strong>
-          </span>
-        </h3>
-        <OrderDetailTable data={orderDetails.items} />
-        <h4>
-          Total Price: <span>{orderDetails.totalAmount}</span>
-        </h4>
+      {orderDetails && (
+        <div className='order-detail'>
+          <h3 className='order-owner'>
+            Order by :
+            <span>
+              <strong>{orderDetails.user.userName}</strong>
+            </span>
+          </h3>
+          <OrderDetailTable data={orderDetails.items} />
+          <h4>
+            Total Price: <span>{orderDetails.totalAmount}</span>
+          </h4>
+          <Steps current={currentStep}>
+            {status.map((status) => (
+              <Step
+                title={status.type}
+                description={<FormatDate date={status.date} />}
+              />
+            ))}
+          </Steps>
 
-        <Steps current={currentStep}>
-          {orderDetails.orderStatus.map((status) => (
-            <Step
-              title={status.type}
-              description={<FormatDate date={status.date} />}
-            />
-          ))}
-        </Steps>
-
-        <div className='order-updateStatus'>
-          <Select
-            defaultValue={
-              orderDetails.status == "completed" ? "Finished" : "Update"
-            }
-            disabled={orderDetails.status == "completed" ? true : false}
-            style={{ width: 120 }}
-            onChange={handleChange}
-          >
-            {orderDetails.orderStatus.map((status) => {
-              if (!status.isCompleted) {
-                return <Option value={status.type}>{status.type}</Option>;
+          <div className='order-updateStatus'>
+            <Select
+              defaultValue={
+                orderDetails.status == "completed" ? "Finished" : "Update"
```

Comment:
```suggestion
                orderDetails.status == "completed" ? "Finished" : "In progress"
```
Ngoài ra thì i18n luôn anh nhé